src/bundle/ui-dev/src/modules/common/dropdown/dropdown.js

```diff
@@ -93,6 +91,10 @@ const Dropdown = ({ value, options, onChange, small }) => {
         )
     }
 
+    if (extraClass) {
+        dropdownClasses[extraClass] = true
```

Comment:
```suggestion
        dropdownClasses[extraClass] = true;
```

Reply:
Why this cannot be in createCssClassNames with `[extraClass]: !!extraClass`?

Reply:
* `[extraClass]: true` with defaultProps `extraClasses: ''`.